docs/release_notes.rst

```diff
@@ -32,6 +32,13 @@ runs on most Windows®, Linux® and macOS® platforms.
 `main Connector
 repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 
+Version 1.2.3
+-----------------
+
+What's New in 1.2.3
+^^^^^^^^^^^^^^^^^^^
+
+*Connector* 1.2.3 is built on `RTI Connext DDS 6.1.2 <https://community.rti.com/documentation/rti-connext-dds-612>`__.
 
 Version 1.2.2
```

Comment:
This is still not right.

The whole section for "Version 1.2.2" will need to be moved under "Previous Releases" section.

If this is too much, leave this file untouch and ask tech writer to handle this change. This is about doc change, not version change.